common/endpoints/endpoints.go

```diff
@@ -6,38 +6,56 @@ import (
 	"io"
 	"log"
 	"net/http"
+	"time"
 
 	"github.com/scootdev/scoot/common/stats"
 )
 
-func RegisterHealthCheck(path string) {
-	http.HandleFunc(path, healthHandler)
+func NewObservabilityServer(addr string, stats stats.StatsReceiver) *ObservabilityServer {
```

Comment:
I would just call this ScootServer to avoid name confusion.  This functionality more closely mimics TwitterServer functionality in the JVM. 

Whereas Observability implies something different at Twitter.  Overloaded terms FTW